src/coordination/coordinator_instance_management_server.cpp

```diff
@@ -29,29 +28,30 @@ auto CreateServerContext(const memgraph::coordination::ManagementServerConfig &c
 // NOTE: The coordinator server doesn't more than 1 processing thread - each replica can
 // have only a single coordinator server. Also, the single-threaded guarantee
 // simplifies the rest of the implementation.
-constexpr auto kCoordinatorServerThreads = 1;
+constexpr auto kCoordInstanceManagementServerThreads = 1;
 
 }  // namespace
 
-CoordinatorServer::CoordinatorServer(const ManagementServerConfig &config)
+CoordinatorInstanceManagementServer::CoordinatorInstanceManagementServer(
+    const CoordinatorInstanceManagementServerConfig &config)
     : rpc_server_context_{CreateServerContext(config)},
-      rpc_server_{config.endpoint, &rpc_server_context_, kCoordinatorServerThreads} {
+      rpc_server_{config.endpoint, &rpc_server_context_, kCoordInstanceManagementServerThreads} {
   rpc_server_.Register<replication_coordination_glue::FrequentHeartbeatRpc>([](auto *req_reader, auto *res_builder) {
-    spdlog::debug("Received FrequentHeartbeatRpc on coordinator server");
+    spdlog::debug("Received FrequentHeartbeatRpc on coordinator instance management server");
```

Comment:
They don't, but I would introduce this, we can talk about it for new PR. My reasoning is that current using of nuRAFTs logic for get peer info is not good, since it takes lock and it can deadlock, so I would like to avoid it